packages/react-tinacms-inline/src/inline-form.tsx

```diff
@@ -44,9 +46,20 @@ export interface InlineFormState {
   setFocussedField(field: string): void
 }
 
-export function InlineForm({ form, children }: InlineFormProps) {
+export interface ExtendedInlineFormState extends InlineFormState {
+  name: string
+}
+
+export function InlineForm({ form, children, onFocusChange, name }: InlineFormProps) {
   const [focussedField, setFocussedField] = React.useState<string>('')
 
+  const updateAndNotifyFocus = (field: string) => {
```

Comment:
How would we go about deselecting blocks from other form instances?

Reply:
Something like, in `lucid-admin`:
```
// State for tracking the currently focussed section
const [currentFocussedName, setCurrentFocussedName] = useState<string | null>(null)

// Function for updating the current focussed section
const handleFocusChange = (name: string, newFocussedField: string) => {
  setCurrentFocussedName(newFocussedField ? name : null);
}

return (
  <>
    {/* ... existing code ... */}
    <header className="header-blocks" ref={headerRef}>
      {/* ... existing code ... */}
      <InlineForm
        max={1}
        className='header-editor inline-editor'
        name='header_blocks'
        form={pageForm}
        blocks={{Navigation: inlineFormBlocks['Navigation']}}
        onFocusChange={(newFocussedField) => handleFocusChange('header_blocks', newFocussedField)}
      />
    </header>
    ...
  </>
)
```

Reply:
I'm having trouble building Tina right now. Let me reach out when I get it working